eng/pipelines/pr.yml

```diff
@@ -14,31 +14,29 @@ jobs:
 
     strategy:
       matrix:
-        # Node 18
-        "Linux - Node 18.x - Dotnet 6":
+        # Node 18 + .NET 8
+        "Linux - Node 18.x + .NET 8":
```

Comment:
I would maybe just omit the fact that we have a version of `.net` in the matrix name as it is not super relevant.

Reply:
this was just probably not cleaned since the split

Reply:
Agreed, removed